utils/prometheus/src/lib.rs

```diff
@@ -33,7 +33,7 @@ use std::net::SocketAddr;
 mod networking;
 mod sourced;
 
-pub use sourced::{SourcedCounter, SourcedGauge, MetricSource};
+pub use sourced::{SourcedCounter, SourcedGauge, MetricSource, SourcedMetric};
```

Comment:
cc @mxinden MetricSource doc link to SourcedMetric, SourcedMetric is public but not reexported in a private module, is it a miss or should the doc of MetricSource be updated ?